packages/adapter-utils/test/utils.test.ts

```diff
@@ -1462,4 +1464,11 @@ describe('Test utils.ts', () => {
       expect(res.value.obj[0].undeployable).toBeUndefined()
     })
   })
+
+  describe('getAllReferenceIds', () => {
```

Comment:
nit: test suite name should match function name
```suggestion
  describe('getAllReferencedIds', () => {
```